include/crocoddyl/core/solvers/ipopt/ipopt.hpp

```diff
@@ -0,0 +1,30 @@
+#ifndef CROCODDYL_CORE_SOLVERS_IPOPT_IPOPT_HPP_
+#define CROCODDYL_CORE_SOLVERS_IPOPT_IPOPT_HPP_
+
+#include "coin-or/IpIpoptApplication.hpp"
+
+#include "crocoddyl/core/solver-base.hpp"
+#include "crocoddyl/core/solvers/ipopt/multiple-shooting-nlp.hpp"
+namespace crocoddyl {
+class SolverIpOpt : public SolverAbstract {
+ public:
+  SolverIpOpt(boost::shared_ptr<crocoddyl::ShootingProblem> problem);
+  ~SolverIpOpt();
+
+  bool solve(const std::vector<Eigen::VectorXd>& init_xs = DEFAULT_VECTOR,
+             const std::vector<Eigen::VectorXd>& init_us = DEFAULT_VECTOR, const std::size_t maxiter = 100,
+             const bool is_feasible = false, const double regInit = 1e-9);
+
+ private:
+  Ipopt::SmartPtr<MultipleShootingNlp> ms_nlp_;
```

Comment:
Renamed this class to `IpoptInterface`, i.e.,
```suggestion
  Ipopt::SmartPtr<IpoptInterface> ipopt_iface_;
```